### PR TITLE
Added current/past education section

### PR DESCRIPTION
**Pull request by MAMV3x3**
I added the education section where there are tabs to select the fellow whose education you want to show, the information is shown in the form of a timeline and the data is captured from a JSON file inside the `data` folder called `education.json` where it can be easily edited.
Current Result:
image

### CODE REVIEW

**Review by a-brown-code — APPROVED** [review]
Great work! 👍

File: app/__init__.py [hunk 1]
```diff
@@ -1,11 +1,20 @@
...
+import json
 
 load_dotenv()
 app = Flask(__name__)
 
 
 @app.route('/')
 def index():
-    return render_template('index.html', title="MLH Fellow", url=os.getenv("URL"))
+    # Get JSON data path
```
Comment (Pradyuman7) [inline app/__init__.py:12]: Same comment as https://github.com/MAMV3x3/MLH-Fellowship-pe-portfolio-site/pull/18#discussion_r1239363846

File: app/static/styles/main.css [hunk 2]
```diff
@@ -95,6 +95,99 @@ body {
...
+    color: #555;
+    font-weight: 300;
+}
+
+.education-section .member-tabs .tab-button {
+    padding: 10px 20px;
+    border: none;
+    background-color: #f5f5f9;
+    color: #1d539f;
+    font-size: 16px;
```
Comment (Pradyuman7) [inline app/static/styles/main.css:174]: `em` can be better here

File: app/templates/index.html [hunk 3]
```diff
@@ -41,5 +41,55 @@
...
+                        <p>{{ education.institution }}</p>
+                    </div>
+                </li>
+                {% endfor %}
+            </ul>
+            {% endfor %}
+        </div>
+    </div>
+
+    <script>
```
Comment (Pradyuman7) [inline app/templates/index.html:73]: Would it be better here to separate this into another js file and call that method? I would think so, but am asking as you might have another reason to do it like this

Comment (MAMV3x3) [reply app/templates/index.html:73]: I think separating it would definitely be better, I'll get to it right away!

**Comment by MAMV3x3** [comment]
I just implemented the improvements, detailed information is in commit: `ID: 7b80e8208956a9576c900ca2d7ad231d002bc00b`

**Review by Pradyuman7 — APPROVED** [review]
LGTM 👍

**Pull request merged**